Bricks/net/http/impl/server.h

```diff
@@ -329,7 +329,17 @@ class GenericHTTPRequestData : public HELPER {
           if (p) {
             *p = '\0';
             const char* const key = &buffer_[current_line_offset];
-            const char* const value = p + constants::kHeaderKeyValueSeparatorLength;
+            const char* value = p + constants::kHeaderKeyValueSeparatorLength;
+
+            // Ignore trailing spaces and tabs before and after the value
+            while (value < next_crlf_ptr && ::isspace(*value)) {
```

Comment:
Don't know for sure, but potentially, `::isspace` may cover more than `SP | HTAB` (e.g. a `VTAB`), so this is not strictly conforming to the RFC.